experiments/zrq/ansible/11-install-hadoop.yml

```diff
@@ -0,0 +1,100 @@
+#
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2020, ROE (http://www.roe.ac.uk/)
+#
+#     This information is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation, either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This information is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#   </meta:licence>
+# </meta:header>
+#
+#
+
+#
+# This assumes we need the same binaries installed on the masters and workers.
+#
+# This uses version hadoop-3.2.1, which isn't listed on the Hadoop wiki ? 
```

Comment:
Any particular reason for using 3.2.1 ? Is this related to the Spark version you are using?

Reply:
Good question - It was the most recent available at the time.

Reply:
Since then, there has been a newer release, 3.3, which claims to solve (some of) the issues with Java 11.
https://cwiki.apache.org/confluence/display/HADOOP/Hadoop+3.3+Release

Reply:
Created a new issue to update the deployment to the newer version (#93).